### ISSUE TITLE
Web authenticator prompt on ios - Can it be avoided being displayed each time

### ISSUE THREAD

**Issue by JRallee** [xamarin/Essentials#1607]
### Description
On iOS each time web authenticator opens browser for authentication (Sign in and Sign out), a prompt is displayed requesting access. Can we avoid this each time?

### Steps to Reproduce

1. Create a Xamarin Forms application
2. Use web authenticator to authenticate (Active directory authentication is configured and working fine)
3. When user signs in or signs out, a prompt is displayed before opening Safari browser in a dialog requesting access.

Can this be avoided after the first time?
Can this be added as an entitlement or something?

**Comment by dfarrNTST** [comment]
Would like to know this as well...it is driving our iOS user's crazy having to tap this every time.

**Comment by dominicpease** [comment]
Yes it can. I have an open proposal at #1528 which should give you a configuration option to solve this. I was waiting for approval (?) on that proposal, but I might just go ahead and put together a PR on it.

**Comment by JRallee** [comment]
> Yes it can. I have an open proposal at #1528 which should give you a configuration option to solve this. I was waiting for approval (?) on that proposal, but I might just go ahead and put together a PR on it.

Nice, this change would be real helpful in terms of the user experience.

**Comment by snehalInfogain** [comment]
@dominicpease Do you have any estimated time for this enhancement to go live? We are integrating this SDK and wanted to go live with this bug resolved asap.

**Comment by Px7-941** [comment]
This is an Apple thing. The developers can't do much about it except give us an option to force SFSafariViewController.
